workers/unity/Packages/com.improbable.gdk.buildsystem/WorkerBuilder.cs

```diff
@@ -118,7 +121,7 @@ public static void BuildWorkerForEnvironment(string workerType, BuildEnvironment
 
         public static void Clean()
         {
-            Directory.Delete(PlayerBuildDirectory, true);
+            Directory.Delete(AssetDatabaseDirectory, true);
```

Comment:
Do we want to not clean the `PlayerBuildDirectory`, if not, you can remove the static string above?

Reply:
`PlayerBuildDirectory` is used in other parts of the class, so we can't remove the static string

We can do the hot swap on this line because `AssetDatabaseDirectory` is the parent of `PlayerBuildDirectory`

Reply:
Ah nice 👍